sawb_custom_hide_vat_for_individual/views/res_partner_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="view_partner_form">
+        <field name="name">view_partner_form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="vat" position="attributes">
+                <attribute
+                    name="attrs"
+                >{'invisible': [("is_company", "=", False)]}</attribute>
```

Comment:
just a rant against odoo: good variable naming and design would make the condition much more understandable: `{'visible': [("is_company", "=", True)]}`. general dev rule: always use positive names for flag attributes.

Reply:
Good point, I always spend a few seconds computing the condition in my head ^^